www/js/transforms/collapseTables.js

```diff
@@ -1,42 +1,6 @@
 var transformer = require("../transformer");
 var utilities = require("../utilities");
-
-/*
-Tries to get an array of table header (TH) contents from a given table.
-If there are no TH elements in the table, an empty array is returned.
-*/
-function getTableHeader( element ) {
-    var thArray = [];
-    if (element.children === undefined || element.children === null) {
-        return thArray;
-    }
-    for (var i = 0; i < element.children.length; i++) {
-        var el = element.children[i];
-        if (el.tagName === "TH") {
-            // ok, we have a TH element!
-            // However, if it contains more than two links, then ignore it, because
-            // it will probably appear weird when rendered as plain text.
-            var aNodes = el.querySelectorAll( "a" );
-            if (aNodes.length < 3) {
-                // Also ignore it if it's identical to the page title.
-                if (el.innerText.length > 0 && el.innerText !== window.pageTitle && el.innerHTML !== window.pageTitle) {
-                    thArray.push(el.innerText);
-                }
-            }
-        }
-        //if it's a table within a table, don't worry about it
-        if (el.tagName === "TABLE") {
-            continue;
-        }
-        //recurse into children of this element
-        var ret = getTableHeader(el);
-        //did we get a list of TH from this child?
-        if (ret.length > 0) {
-            thArray = thArray.concat(ret);
-        }
-    }
-    return thArray;
-}
+var getTableHeader = require("applib").CollapseElement.getTableHeader;
```

Comment:
`getTableHeader` was the first method we moved to the common [applib](https://github.com/wikimedia/applib) repo 

We then removed `getTableHeader` from our `www/js/transforms/collapseTables.js` iOS source file, then ran `make grunt` to rebuild the bundled `assets` folder contents and confirmed the version we removed disappeared from `Wikipedia/assets/index.js` and the version we added to [applib](https://github.com/wikimedia/applib) appeared in `Wikipedia/assets/index.js`.